README.md

```diff
@@ -41,7 +45,11 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- Replace this comment with your answer -->
+x = 4
+y = 3
+x = 4 : y= 4
```

Comment:
Not quite. `x = y` takes y's value and copies it into x. As of this line, y is equal to 3, so that means that x will be set to 3.